ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplDumpTask.java

```diff
@@ -414,7 +441,8 @@ private boolean shouldDump(Path previousDumpPath) throws IOException {
       return true;
     } else {
       FileSystem fs = previousDumpPath.getFileSystem(conf);
-      return fs.exists(new Path(previousDumpPath, LOAD_ACKNOWLEDGEMENT.toString()));
+      return fs.exists(new Path(previousDumpPath, LOAD_ACKNOWLEDGEMENT.toString()))
+              && !fs.exists(new Path(previousDumpPath, FAILOVER_READY_MARKER.toString()));
```

Comment:
If the skip is because of failover, we should also also ensure if there is no open write txn at this point of time and if found one, we should fail the policy with irrecoverable error.